pkgs/applications/misc/fuzzel/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchgit, pkgconfig, meson, ninja, wayland, pixman, cairo, librsvg, wayland-protocols, wlroots, libxkbcommon, scdoc, git }:
+
+stdenv.mkDerivation rec {
+  pname = "fuzzel";
+  version = "1.3.0";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/fuzzel";
+    rev = "${version}";
+    sha256 = "12jv5iwmksygw8nfkxbd9rbi03wnpgb30hczq009aqgy7lyi5zmp";
+  };
+
+  tllist = fetchgit {
+    url = "https://codeberg.org/dnkl/tllist.git";
+    rev = "cc0988be9c131ae4b34f7c4e748fce5f468a9f04";
+    sha256 = "06gzq5iafc2jc7w765xrav1sdaqqsy76swyas941bryb07m8rmwq";
+  };
+
+  fcft = fetchgit {
+    url = "https://codeberg.org/dnkl/fcft.git";
+    rev = "0.4.3";
+    sha256 = "1r2k5726k6ps8ml2s1vqmpiggqxzq9pbzs7m0dsxk29mh8vg0psj";
+  };
```

Comment:
Why not package these as separate packages?

Reply:
I didnt found anything else that uses these libraries. But sure can do